bonelab/cli/RapidPrototypeLetters.py

```diff
@@ -0,0 +1,487 @@
+  transformPost1.SetTransform( transform1 )
+  transformPost1.Update()
+  
+  transformPost2 = vtk.vtkTransformFilter()
+  transformPost2.SetInputConnection( transformPost.GetOutputPort() )
+  transformPost2.SetTransform( transform2 )
+  transformPost2.Update()
+  
+  transformPost3 = vtk.vtkTransformFilter()
+  transformPost3.SetInputConnection( transformPost.GetOutputPort() )
+  transformPost3.SetTransform( transform3 )
+  transformPost3.Update()
+  
+  transformPost4 = vtk.vtkTransformFilter()
+  transformPost4.SetInputConnection( transformPost.GetOutputPort() )
+  transformPost4.SetTransform( transform4 )
+  transformPost4.Update()
+  
+  post12 = JoinPolyData(transformPost1.GetOutput(),transformPost2.GetOutput())
+  post123 = JoinPolyData(post12.GetOutput(),transformPost3.GetOutput())
+  post1234 = JoinPolyData(post123.GetOutput(),transformPost4.GetOutput())
+  post1234.Update()
+  
+  cube_and_posts = JoinPolyData(cube.GetOutput(),post1234.GetOutput())
+  
+  mat4x4 = visualize_actors( cube_and_posts.GetOutputPort(), bone1234.GetOutputPort()  )
+  #mat4x4 = visualize_actors( cube_and_posts.GetOutputPort(), None  )
+  
+  #ofile = "/Users/skboyd/Documents/projects/FormLabs/radius_time_series/bone_mask_post_and_table.stl" 
+  #message('Writing STL file: '+ofile)
+  #writer = vtk.vtkSTLWriter()
+  #writer.SetFileName(ofile)
+  #writer.SetFileTypeToASCII()
+  #writer.SetInputConnection( cube_and_posts.GetOutputPort() )
+  #writer.Write()
+  
+  ofile = "/Users/skboyd/Documents/projects/FormLabs/radius_time_series/bone_table.aim" 
+  el_size_mm = [0.0760,0.0760,0.0760]
+  
+  message('Writing AIM file: '+ofile)
+  whiteImage = vtk.vtkImageData()
+  whiteImage.SetSpacing(el_size_mm)
+  bounds = cube_and_posts.GetOutput().GetBounds()
+  dim = [1,1,1]
+  for i in range(3):
+    dim[i] = (math.ceil((bounds[i*2+1]-bounds[i*2]) / el_size_mm[i]))
+  whiteImage.SetDimensions(dim)
+  whiteImage.SetExtent(0, dim[0] - 1, 0, dim[1] - 1, 0, dim[2] - 1)
+  origin = [1,1,1]
+  origin[0] = bounds[0] + el_size_mm[0] / 2
+  origin[1] = bounds[2] + el_size_mm[1] / 2
+  origin[2] = bounds[4] + el_size_mm[2] / 2
+  whiteImage.SetOrigin(origin)
+  whiteImage.AllocateScalars(vtk.VTK_CHAR,1)
+  for i in range(whiteImage.GetNumberOfPoints()):
+    whiteImage.GetPointData().GetScalars().SetTuple1(i, 127)
+  pol2stenc = vtk.vtkPolyDataToImageStencil()
+  pol2stenc.SetInputData( cube_and_posts.GetOutput() )
+  pol2stenc.SetOutputOrigin( origin )
+  pol2stenc.SetOutputSpacing( el_size_mm )
+  pol2stenc.SetOutputWholeExtent( whiteImage.GetExtent() )
+  pol2stenc.Update()
+  imgstenc = vtk.vtkImageStencil()
+  imgstenc.SetInputData(whiteImage)
+  imgstenc.SetStencilConnection( pol2stenc.GetOutputPort() )
+  imgstenc.ReverseStencilOff()
+  imgstenc.SetBackgroundValue(0)
+  imgstenc.Update()
+  writer = vtkbone.vtkboneAIMWriter()
+  writer.SetInputData( imgstenc.GetOutput() )
+  writer.SetFileName(ofile)
+  writer.Update()
+  
+  exit()
```

Comment:
Best to use `return` instead of `exit` in a function
```suggestion
  return
```